web/src/auth/mfa/MfaAuthVerifyForm.js

```diff
@@ -24,7 +24,14 @@ import MfaVerifyTotpForm from "./MfaVerifyTotpForm";
 export const NextMfa = "NextMfa";
 export const RequiredMfa = "RequiredMfa";
 
-export function MfaAuthVerifyForm({formValues, authParams, mfaProps, application, onSuccess, onFail}) {
+export function MfaAuthVerifyForm({
```

Comment:
plz do not change code format

Reply:
@dacongda These seems to be a commit hook configured in repo which runs `yarn run fix` causing these changes. I will reset these changes in the last commit though.